appinfo/info.xml

```diff
@@ -8,9 +8,14 @@
 	<dependencies>
 		<owncloud min-version="9.0" max-version="9.1" />
 	</dependencies>
+
 	<description>Calendar GUI for ownCloud's integrated CalDAV server</description>
 	<documentation>
 		<user>http://doc.owncloud.org/server/9.0/user_manual/pim/calendar.html</user>
 	</documentation>
+	<category>productivity</category>
```

Comment:
Are this properties documented somewhere?
I've never seem them in other apps or anywhere in core.